Game.java

```diff
@@ -136,16 +255,40 @@ public String getInfo(Game newGame){
 	}*/
 }
 
-class bigPlayer{
+class Map{
+	String mapName = "";
+
+	Map(){
+	}
+
+	Map(String map){
+		this.mapName = map;
+	}
+
+	public String getMapName(){
+		return this.mapName;
+	}
+}
+
+class Player{
 	String name = "";
 	boolean isCaptain = false;
```

Comment:
I dont think captain should be a property of a player, it should be property of a lobby which means you probably need a lobby class

Reply:
Also isnt the convention to have members created/assigned in the constructor? it is for C++ at least
@mrvernonliu

Reply:
Can you explain why you think this? I don't see why it would be so different or why it would affect much down the road.

Reply:
I looked this up and it doesn't seem like a big deal: https://stackoverflow.com/questions/15729277/instantiating-an-object-outside-the-methods-or-construcor

But in C++ at least you have to be really explicit about how objects' members get created/instantiated methods because of a whole host of reasons which I will not go into here